nbs/getting_started.ipynb

```diff
@@ -28,21 +28,21 @@
     "\n",
```

Comment:
<p>I thought the word "source code" was more informative than the word "plain text". what was the thinking behind getting rid of the word "source code"?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/925/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>